.github/workflows/bench.yml

```diff
@@ -0,0 +1,38 @@
+name: Benchmarks
```

Comment:
I did not review it; I just glanced over it.

I wasn't expecting this to grow roots into existent packages.

Could we have an `internal/benchmarks` package where we isolate everything about it?

Reply:
Good suggestion @arboleya